### PR TITLE
Fix distancesFromManeuver not being cleared

### LINKED ISSUE #760: Rerouting loop issue - refers to #603, still a bug
Hi,

Version of android-navigation: 0.11.0.
This issue is reproductible in driving test and in simulation test.
You can watch the video here : https://vimeo.com/260241324 .

Even if I change parameters in MapboxNavigationOptions, this bug still appears.
Even if : 
```java
   @Override
    public boolean allowRerouteFrom(Point offRoutePoint) {
        Log.e(TAG, "allowRerouteFrom " + offRoutePoint);
        return false;
    }
```
rerouting occurs.

Coordinates to test it : 
 - departure : 48.890246, 2.247127
 - arrival : 48.888416, 2.244859

### PR DESCRIPTION
**Pull request by danesfeder**
Closes #760

### CODE REVIEW

File: app/src/main/java/com/mapbox/services/android/navigation/testapp/activity/navigationui/NavigationViewActivity.java [hunk 1]
```diff
@@ -265,14 +269,16 @@ private void initLocationLayer() {
   }
 
   private void initMapRoute() {
-    mapRoute = new NavigationMapRoute(mapView, mapboxMap, "admin-3-4-boundaries-bg");
+    mapRoute = new NavigationMapRoute(mapView, mapboxMap);
     mapRoute.setOnRouteSelectionChangeListener(this);
   }
 
   private void fetchRoute() {
+    Point origin = Point.fromLngLat(2.247127, 48.890246);
```
Comment (devotaaabel) [inline app/src/main/java/com/mapbox/services/android/navigation/testapp/activity/navigationui/NavigationViewActivity.java]: Is this supposed to still be hardcoded?

File: app/src/main/java/com/mapbox/services/android/navigation/testapp/activity/navigationui/NavigationViewActivity.java [hunk 2]
```diff
@@ -355,7 +361,9 @@ public void boundCameraToRoute() {
       if (bboxPoints.size() > 1) {
         try {
           LatLngBounds bounds = new LatLngBounds.Builder().includes(bboxPoints).build();
-          animateCameraBbox(bounds, CAMERA_ANIMATION_DURATION, new int[] {50, 500, 50, 335});
+          // left, top, right, bottom
+          int topPadding = launchBtnFrame.getHeight() * 2;
```
Comment (devotaaabel) [inline app/src/main/java/com/mapbox/services/android/navigation/testapp/activity/navigationui/NavigationViewActivity.java]: Here too?

Comment (danesfeder) [reply app/src/main/java/com/mapbox/services/android/navigation/testapp/activity/navigationui/NavigationViewActivity.java]: Yeah this is just to account for the top padding so the route doesn't show up under the launch button

**Comment by danesfeder** [comment]
ezgif com-video-to-gif

**Pull request merged**